assets/scss/layouts/header/_header.scss

```diff
@@ -132,6 +131,16 @@
     }
 }
 
+.header-logo-position-left {
+    float: left;
+    margin-left: remCalc(40px);
+}
+
+.header-logo-position-right {
+    float: right;
+    margin-right: remCalc(40px);
+}
+
```

Comment:
the `margin-right` needs to be adjusted for the mobile breakpoint

![screen shot 2016-04-21 at 1 22 27 pm](https://cloud.githubusercontent.com/assets/7851919/14723092/266e2f24-07c4-11e6-803d-4d706053b54f.png)